feat: two-column tree view

tom-anders opened the pull request:
[twoColumn.webm](https://github.com/user-attachments/assets/a730b7cb-b62b-455a-9dce-c56d7862e7a0)

veloce commented:
@tom-anders really cool, thanks!

I'd suggest an improvement (maybe): we could try to have a full width background color for the non-inline moves (including the move index and padding).
Like what the website does actually. I think that way it would look even better.

tom-anders commented:
> @tom-anders really cool, thanks!
> 
> I'd suggest an improvement (maybe): we could try to have a full width background color for the non-inline moves (including the move index and padding). Like what the website does actually. I think that way it would look even better.

Sure, I’ll try it out. Do we want to use the new background for the old inline notation view as well then? To be consistent?

tom-anders commented:
> Did your originally analysis notation refactoring from a bit earlier prevent stray ( from sitting at end of line. And if so should that carry over here as well. It's decently minor but still slightly odd to see. Not sure if you have any thoughts to have some logic for it

Not 100% sure if we already have logic for that or not, but I don't think so...? Can you open a separate issue for that?

tom-anders commented:
Okay, adjusted the background and also added some horizontal dividers:

Dark Theme:
![Screenshot_1739224202](https://github.com/user-attachments/assets/23eb1cad-0f94-416d-bc61-00c01b3509dc)

Light Theme:
![Screenshot_1739224246](https://github.com/user-attachments/assets/77948a33-cbfd-44c7-878e-7edcdae39cb2)

Website for comparison:

![Screenshot_2025-02-10_22-51-34](https://github.com/user-attachments/assets/f77b7f3f-8086-4821-a7f1-22ff0afeac01)
![Screenshot_2025-02-10_22-51-52](https://github.com/user-attachments/assets/0ad81254-ae84-47a7-bc02-5ddeb75de660)
